src/components/GameMode/index.js

```diff
@@ -0,0 +1,26 @@
+export class GameMode {
+  constructor(config) {
+    this.config = config;
+  }
+
+  changeGameMode() {
+    const gameModeItems = document.querySelectorAll(`.menu__item`);
+    gameModeItems.forEach((modeItem) => {
+      modeItem.addEventListener(`click`, (e) => {
+        this.changeGameModeClassList(e), this.setSelectedGameMode();
+      });
+    });
+  }
+
+  changeGameModeClassList(e) {
+    const gameMode = document.querySelector('.menu__item--selected');
+    gameMode.classList.remove('menu__item--selected');
+    e.target.classList.add('menu__item--selected');
+  }
+
+  setSelectedGameMode() {
+    this.config.selectedGame = document
```

Comment:
Tą wartość możemy odczytać z e.target.innerText i przekazać sobie do tej metody. Nie jest to jakiś wielki problem ale jeżeli mamy jakąś informacje w JS to nie ma co obarczać przeglądarki dodatkową robotą a i kod będzie krótszy :D